lisa/tools/kdump.py

```diff
@@ -451,6 +454,10 @@ def check_crashkernel_loaded(self, crashkernel_memory: str) -> None:
         # Check if memory is reserved for crash kernel
         self._check_crashkernel_memory_reserved()
 
+    def print_additional_info_before_panic(self) -> None:
```

Comment:
It can be a private method of Mariner, and always print it out. There is no shared logic, so a method on super class is not needed.

Reply:
I'm sorry that I couldn't understand this comment.

I want `print_additional_info_before_panic` to be called just before the panic is induced. The panic is induced in the object of `node` not the object of `kdump`. So, calling a private function of `kdump` may not be possible in the call-chain. So, if I remove the method from super class, it may throw exception when other distros are being tested.

Please let me know if my understanding is wrong.

Reply:
I double checked the current implementation of kdump, it looks need refactor to make it clearer. So far, your way may be the best way to implement it. Please use the method name like `capture_info`.

Reply:
Updated the name of the function